src/voice/streamer.rs

```diff
@@ -318,6 +318,8 @@ pub fn ytdl(uri: &str) -> Result<Box<dyn AudioSource>> {
     Ok(pcm(true, ChildContainer(ffmpeg)))
 }
 
+/// Creates a streamed audio source from youtube search results with `youtube-dl`,`ffmpeg` and `ytsearch`
```

Comment:
Missing fullstop at the end of the sentence.
Consider adding a comma before the "and".